### PR TITLE
LeftNavBar

### PR DESCRIPTION
**Pull request by sanyam007**

### CODE REVIEW

File: ats-web/package-lock.json [hunk 1]
```diff
@@ -0,0 +1,15527 @@
+{
```
Comment (i-mahesh) [inline ats-web/package-lock.json]: Why is this file required  for this commit? Remove this file if not required

File: ats-web/src/components/shared/NavBar.js [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+import 'antd/dist/antd.css';
+import Home from '../Home';
+import Candidates from '../Candidates';
+import Positions from '../Positions';
+import Schedules from '../Schedules';
+import { BrowserRouter as Router, Switch, Route, NavLink } from "react-router-dom";
+const { Header, Sider, Content } = Layout;
+
+class NavBar extends React.Component {
+    siderWidth = "200px";
```
Comment (i-mahesh) [inline ats-web/src/components/shared/NavBar.js]: Use CSS classes instead

File: ats-web/src/components/shared/NavBar.js [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+    collapsableWidth = "80px";
```
Comment (i-mahesh) [inline ats-web/src/components/shared/NavBar.js]: Use CSS classes instead

File: ats-web/src/components/shared/NavBar.js [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+
+
+    render() {
+        const { match, location, history } = this.props;
+
+        return (
+            <Router>
+                <div className="row" >
+                    <Layout>
+                        <div style={{ width: this.state.collapsed ? this.collapsableWidth : this.siderWidth, transition: "all 0.2s" }}>
```
Comment (i-mahesh) [inline ats-web/src/components/shared/NavBar.js]: Use CSS classes instead of inline styles

File: ats-web/src/components/shared/NavBar.js [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+                            <Sider collapsible collapsed={this.state.collapsed} onCollapse={this.toggle}
+                                collapsible={true} width={this.siderWidth} collapsedWidth={this.collapsableWidth} style={{ height: "100vh", position: "fixed", left: 0 }}>
+                                <div className="logo" >
+                                    <img src={this.state.showImage} className={this.state.imgCss} />
+                                </div>
+
+                                <Menu theme="dark" mode="inline" selectedkeys={[]} selectable={false}>
+                                    <Menu.Item key="1">
+                                        <NavLink to="/" exact activeStyle={{ color: "yellow" }}>
```
Comment (i-mahesh) [inline ats-web/src/components/shared/NavBar.js]: Use CSS classes instead of inline styles

File: ats-web/src/components/shared/NavBar.js [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+                                            <Icon type="home" />
+                                            <span>Home</span>
+                                        </NavLink>
+                                    </Menu.Item>
+                                    <Menu.Item key="2">
+                                        <NavLink to="/positions" exact activeStyle={{ color: "yellow" }}>
```
Comment (i-mahesh) [inline ats-web/src/components/shared/NavBar.js]: Use CSS classes instead of inline styles

File: ats-web/src/components/shared/NavBar.js [hunk 7]
```diff
@@ -0,0 +1,112 @@
...
+                                            <Icon type="idcard" />
+                                            <span>Positions</span>
+                                        </NavLink>
+                                    </Menu.Item>
+
+                                    <Menu.Item key="3">
+                                        <NavLink to="/candidates" exact activeStyle={{ color: "yellow" }}>
```
Comment (i-mahesh) [inline ats-web/src/components/shared/NavBar.js]: Use CSS classes instead of inline styles

File: ats-web/src/components/shared/NavBar.js [hunk 8]
```diff
@@ -0,0 +1,112 @@
...
+                                            <Icon type="user" />
+                                            <span>Candidates</span>
+                                        </NavLink>
+                                    </Menu.Item>
+                                    <Menu.Item key="4">
+                                        <NavLink to="/schedules" exact activeStyle={{ color: "yellow" }}>
```
Comment (i-mahesh) [inline ats-web/src/components/shared/NavBar.js]: Use CSS classes instead of inline styles

File: ats-web/src/components/shared/NavBar.js [hunk 9]
```diff
@@ -0,0 +1,112 @@
...
+                                            <span>Schedules</span>
+                                        </NavLink>
+                                    </Menu.Item>
+
+                                </Menu>
+                            </Sider>
+                        </div>
+                        <div className="col-md-10">
+                            <Layout>
+                                <Content style={{ margin: "20px" }} >
```
Comment (i-mahesh) [inline ats-web/src/components/shared/NavBar.js]: Use CSS classes instead of inline styles

File: ats-web/src/styles/shared.css [hunk 10]
```diff
@@ -2,4 +2,81 @@
...
+      height: 100vh;
+  }
+  .margin-20{
+      margin: 20px 20px;
+  }
+  .ant-menu-item a {
+    text-decoration: none;
+  }
+  .ant-menu-item {
+      padding-left: 12px!important;
```
Comment (aparna-salokhe) [inline ats-web/src/styles/shared.css]: avoid important rule
